test/stdlib/test_helper.rb

```diff
@@ -81,7 +81,13 @@ def wrapped_object
                       end
 
           begin
-            return_value = spy.object.__send__(spy.method_name, *args, &spy_block)
+            if spy_block
+              return_value = spy.object.__send__(spy.method_name, *args) do |*a, **k, &b|
+                spy_block.call(*a, **k, &b)
+              end
+            else
+              return_value = spy.object.__send__(spy.method_name, *args, &spy_block)
+            end
```

Comment:
This to go around the fact that [the ruby source](https://github.com/ruby/ruby/blob/master/eval.c#L1398) seems to require the block given to `refine` have to be a proper block a not a `Proc`.

Reply:
Thanks.

I would skip the test rather than adding this code, but we can do that when this actually causes a problem. 💪